src/components/OutsideClickHandler.jsx

```diff
@@ -18,21 +19,18 @@ export default class OutsideClickHandler extends React.Component {
   }
 
   componentDidMount() {
-    if (document.addEventListener) {
-      // `useCapture` flag is set to true so that a `stopPropagation` in the children will
-      // not prevent all outside click handlers from firing - maja
-      document.addEventListener('click', this.onOutsideClick, true);
-    } else {
-      document.attachEvent('onclick', this.onOutsideClick);
-    }
+    // `capture` flag is set to true so that a `stopPropagation` in the children
+    // will not prevent all outside click handlers from firing - maja
+    this.clickHandle = addEventListener(
```

Comment:
Yeah we don't actually export it in `index.js`. Also `attachEvent` is a relic of back when we cared about IE8.